app/views/players/_form.haml

```diff
@@ -7,6 +7,10 @@
   = f.number_field :jersey_number, min: 0, max: 99
   = f.text_field :jersey_name
   = f.text_field :nickname
+  = f.select :team_id,
```

Comment:
managing which players belong to which teams should be an admin-only action. because this controls e.g. who is invited to rsvp to events by default. important information for admin would be:
- how old is the player (allowed to play in Ü30 team as an example)
- does he have a player pass (allowed to play in League and CupMatches)